test/runtests.jl

```diff
@@ -1,33 +1,53 @@
 using Oscar
 using Test
-using Aqua
 using Documenter
+using Distributed
 
 import Random
 
+if !isempty(ARGS)
+  jargs = [arg for arg in ARGS if startswith(arg, "-j")]
+  if !isempty(jargs)
+    numprocs = parse(Int,split(jargs[end], "-j")[end])
+  end
+elseif haskey(ENV, "NUMPROCS")
+  numprocs = parse(Int,ENV["NUMPROCS"])
+else
+  numprocs = 1
+end
```

Comment:
So if `ARGS` is non-empty but contains no `-j` flag, then `numprocs` is left uninitialized...

Here is an alternative that avoids this.
```suggestion
numprocs_str = get(ENV, "NUMPROCS", "1")
if !isempty(ARGS)
  jargs = [arg for arg in ARGS if startswith(arg, "-j")]
  if !isempty(jargs)
    numprocs_str = split(jargs[end], "-j")[end]
  end
end

const numprocs = parse(Int, numprocs_str)
```